docs/source/manim_directive.py

```diff
@@ -53,29 +140,38 @@ def run(self):
         source_block = '\n'.join(source_block)
         
         file_writer_config_code = [
+            f'config["frame_rate"] = {frame_rate}',
+            f'config["pixel_height"] = {pixel_height}',
+            f'config["pixel_width"] = {pixel_width}',
             'file_writer_config["media_dir"] = "./source/media"',
             'file_writer_config["images_dir"] = "./source/media/images"',
             'file_writer_config["video_dir"] = "./source/media/videos"',
             f'file_writer_config["save_last_frame"] = {save_last_frame}',
             f'file_writer_config["save_as_gif"] = {save_as_gif}'
         ]
+
+        user_code = self.content
+        if user_code[0].startswith('>>> '): # check whether block comes from doctest
+            user_code = [line[4:] for line in user_code 
+                         if line.startswith(('>>> ', '... '))]
+
         code = [
-            'from manim import *', 
-            *file_writer_config_code, 
-            *self.content, 
+            'from manim import *',
+            *file_writer_config_code,
+            *user_code,
             f'{clsname}()'
         ]
         exec('\n'.join(code), globals())
 
         # copy video file to output directory
         if not (save_as_gif or save_last_frame):
             filename = f'{clsname}.mp4'
-            filesrc = f'source/media/videos/1080p60/{filename}'
+            filesrc = f'source/media/videos/{qualitydir}/{filename}'
```

Comment:
Comment: once #337 is closed, this will have to change. Let's make a note to come back to this